### PR TITLE
continued the matrix thread

### PR DESCRIPTION
**Pull request by mgvalentin**

### CODE REVIEW

**Comment by mgvalentin** [comment]
Please review this, @casey-collab

File: english/matrix/matrix.md [hunk 1]
```diff
@@ -22,3 +22,6 @@ belly and see the outside world.
 It's dark and fluffy. 
 While you don't believe that you might be the One,
  it's only up to you to save the world.
+
+[You crawl through the hole you just made and...](english/matrix/matrix2.md)
```
Comment (casey-collab) [inline english/matrix/matrix.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/matrix/matrix2.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (casey-collab) [inline english/matrix/matrix2.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/matrix/matrix2.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**